pyproject.toml

```diff
@@ -39,7 +39,7 @@ exclude = [
 
 [tool.poetry.dependencies]
 python = "~3.12"
-django = "^4.2"
```

Comment:
Similar to the above comment, we should limit this to what we can support - `Django >4.2,<5.1 compatibility`

Reply:
Done